Dockerfile

```diff
@@ -15,8 +15,14 @@ RUN apk --no-cache --update add \
     gem install oj && \
     gem install json && \
     gem install fluentd -v 0.12.29 && \
+    mkdir -p /tmp/jemalloc && \
```

Comment:
Don't you use this directory? The first design looks:
* make a temp dir `/tmp/jemalloc` to download & extract jemalloc tarball
* configure && make && install it
* remove `/tmp/jemalloc`

But actually, this code uses `/`.